x-pack/platform/plugins/shared/fleet/server/tasks/sync_integrations/get_remote_status.ts

```diff
@@ -63,14 +60,30 @@ export const getRemoteSyncedIntegrationsInfoByOutputId = async (
       method: 'GET',
     };
     const url = `${kibanaUrl}/api/fleet/remote_synced_integrations/status`;
-    logger.info(`Fetching ${kibanaUrl}/api/fleet/remote_synced_integrations/status`);
+    logger.debug(`Fetching ${kibanaUrl}/api/fleet/remote_synced_integrations/status`);
 
+    let body;
+    let errorMessage;
     const res = await fetch(url, options);
+    try {
+      body = await res.json();
+    } catch (error) {
+      errorMessage = `GET ${url} failed with status ${res.status}. ${error.message}`;
+    }
 
-    const body = await res.json();
+    if (body?.statusCode && body?.message) {
+      errorMessage = `GET ${url} failed with status ${body.statusCode}. ${body.message}`;
+    }
 
-    return body as GetRemoteSyncedIntegrationsStatusResponse;
+    return {
+      integrations: body?.integrations ?? [],
+      custom_assets: body?.custom_assets,
+      error: errorMessage ?? body?.error,
```

Comment:
Returning error from remote API, e.g. when follower index is paused
<img width="607" alt="image" src="https://github.com/user-attachments/assets/ab6fea8c-15f9-4f3e-9ef9-0f4856c0c685" />